next/oracle/identity_db_cleanup.sh

```
@@ -0,0 +1,126 @@
+        DELETE_PROCEDURE+="DELETE FROM $table WHERE INVITED_ORG_ID = '$ORG_UUID';"
+      else
+        DELETE_PROCEDURE+="DELETE FROM $table WHERE TENANT_ID = $TENANT_ID;"
+      fi
+    done
+
+    # Execute the delete procedure
+    echo "Executing deletion procedure on identity database for TENANT_ID=$TENANT_ID..."
+    echo "$DELETE_PROCEDURE" | sqlplus -s "$DB_USER/$DB_PASSWORD@//$DB_HOST:$DB_PORT/$DB_NAME" > /dev/null 2>&1
+
+    # Check if the execution was successful
+    if [[ $? -eq 0 ]]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)